syfertext/pointers/token_pointer.py

```diff
@@ -0,0 +1,50 @@
+from syft.generic.pointers.object_pointer import ObjectPointer
+from syft.workers.base import BaseWorker
+import syft as sy
+
+from typing import List
+from typing import Union
+
+
+class TokenPointer(ObjectPointer):
+    def __init__(
+        self,
+        location: BaseWorker = None,
+        id_at_location: Union[str, int] = None,
+        owner: BaseWorker = None,
+        id: Union[str, int] = None,
+        garbage_collect_data: bool = True,
+        tags: List[str] = None,
+        description: str = None,
+    ):
+
+        super(TokenPointer, self).__init__(
+            location=location,
+            id_at_location=id_at_location,
+            owner=owner,
+            id=id,
+            garbage_collect_data=garbage_collect_data,
+            tags=tags,
+            description=description,
+        )
+
+    @property
+    def text(self):
+
```

Comment:
Actually I had a doubt in this. It actually returns a `PointerTensor` for some reason. I thought Syft's String could be recieved in this way as `int` is recieved in `__len__` . Could you help in this ?

Reply:
Can you do an experiment, in Token, make the `text_` method return a syft String , I think this way, you will see a StringPointer returned here

Reply:
the `text_` method in `Token` does return a `syft String` in this commit . Still it is returning a `PointerTensor`.
But maybe we can do one thing, I can register the `syft String` on the owner of `Token` and then create a `StringPointer` manually in `text_` of `TokenPointer`

Reply:
Ok, try this out and let's see with use if it will be convenient

Reply:
For doing this, we will need to add `owner` and `client_id` attributes to the `Token` object.
Only then, the `text_()` method will know if it has to return a `syft.String` or `id` to create `syft.StringPointer`.

Reply:
I changed the `text_()` to `string()`. As it was the cause of the issue of getting back a `PointerTensor` rather than an `int`, while returning the `string.id`.